app/templates/admin/content/translations.hbs

```diff
@@ -16,7 +16,7 @@
               {{get l10n.availableLocales language.code}}
             </div>
             <div class="column eight wide">
-              <a href="{{language.sourceUrl}}">{{t 'Download'}}</a>
+              <a href="{{language.sourceUrl}}" onclick={{action 'translationsDownload'}}>{{t 'Download' }}</a>
```

Comment:
No need to use onclick. It will call it automatically. Also make it a button, not an anchor tag.